pkg/collector/corechecks/oracle/config/config.go

```diff
@@ -0,0 +1,85 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package config
+
+import (
+	"fmt"
+
+	"gopkg.in/yaml.v3"
+
+	"github.com/DataDog/datadog-agent/pkg/autodiscovery/integration"
+	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/oracle/common"
+	"github.com/DataDog/datadog-agent/pkg/obfuscate"
+)
+
+// InitConfig is used to deserialize integration init config.
+type InitConfig struct {
+	GlobalCustomQueries   []MetricConfig `yaml:"global_custom_metrics"`
+	Service               string         `yaml:"service"`
+	MinCollectionInterval int            `yaml:"min_collection_interval"`
+}
+
+// InstanceConfig is used to deserialize integration instance config.
+type InstanceConfig struct {
+	Server                 string              `yaml:"server"`
+	Port                   int                 `yaml:"port"`
+	ServiceName            string              `yaml:"service_name"`
+	Protocol               string              `yaml:"protocol"`
+	Username               string              `yaml:"username"`
+	Password               string              `yaml:"password"`
+	TnsAlias               string              `yaml:"tns_alias"`
+	TnsAdmin               string              `yaml:"tns_admin"`
+	DBM                    bool                `yaml:"dbm"`
+	Tags                   []string            `yaml:"tags"`
+	ObfuscatorOn           bool                `yaml:"obfuscator_on"`
```

Comment:
Because it's safer to have the obfuscator enabled for data privacy reasons, I think it would make sense to flip this configuration option around. Instead of a config to _enable_ obfuscation, it should be a config to _allow_ unobfuscated queries. 

So maybe name it `AllowUnobfuscatedStatements` (or something along those lines)?